mini-modeline.el

```diff
@@ -94,6 +94,10 @@ Nil means current selected frame."
   :type 'sexp
   :group 'mini-modeline)
 
+(defcustom mini-modeline-right-padding 3
+  "Padding to use in the right side.
+Set this to the minimal value that doesn't cause truncation.")
```

Comment:
Can you add `:type` and `:group` to this `defcustom`? So user can easily find this via UI

Reply:
Done.